vowpalwabbit/slim/include/vw_slim_return_codes.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#define S_VW_PREDICT_OK									0
```

Comment:
Replace tabs with spaces?

Reply:
I want to keep this as close to the code base to original as possible to facilitate 3rd party integration.

---
In reply to: [270508197](https://github.com/VowpalWabbit/vowpal_wabbit/pull/1800#discussion_r270508197) [](ancestors = 270508197)